lib/iris/tests/unit/plot/test__get_plot_objects.py

```diff
@@ -0,0 +1,29 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""Unit tests for the `iris.plot._get_plot_objects` function."""
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests  # isort:skip
+
+import iris.cube
+
+if tests.MPL_AVAILABLE:
+    from iris.plot import _get_plot_objects
+
+
+@tests.skip_plot
+class Test_get_plot_objects(tests.IrisTest):
```

Comment:
Should this have an extra `_` after `Test` for naming consistency?

Reply:
I copied from another file in this directory but, now that I look at a few, I see there is no consistency.  So agreed.  Done.